packages/experimental/RadioGroup/src/Radio/Radio.types.ts

```diff
@@ -76,7 +97,12 @@ export interface RadioProps extends IPressableProps {
   enableFocusRing?: boolean;
 }
 
-export interface RadioState extends IPressableHooks<RadioProps & React.ComponentPropsWithRef<any>> {}
+export interface RadioState extends IPressableHooks<RadioProps & React.ComponentPropsWithRef<any>> {
+  /**
+   * If horizontal-stacked, label should be below indicator.
+   */
+  labelPositionBelow?: boolean;
```

Comment:
Nit this isn't necessarily required - you could change the lookup function to check for the labelPositionBelow state by looking at the props

Reply:
Moved logic to lookup function